index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <!-- Google tag (gtag.js) -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-91FC3FP4RX"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', 'G-91FC3FP4RX');
+  </script>
+  <!-- End Google Tag Manager -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style/reset.css">
+  <link rel="stylesheet" href="./style/style.css">
+  <link rel="stylesheet" href="./style/intro.css">
+  <title>판다마켓</title>
+</head>
+<body>
+  <header>
+    <nav>  
+      <h1 id="title"><a href="/"><img src="./img/title.png" alt="판다마켓 타이틀"></a></h1>
+      <div id="nav-right">
+        <a id="btn-login" class="s-btn" href="/login" >로그인</a>
+      </div>
+    </nav>
+  </header>
+  <div id="header-gap"></div> <!--fixed된 header와 아래 다른 요소의 위치 보정을 위한 임시적 갭-->
+  <main id="intro-big-box">
+    <section class="intro-top">
+      <div class="intro-tobo-item">
+        <div class="tobo-text">
+          <h2>일상의 모든 물건을 <br>거래해 보세요</h2>
+          <a class="l-btn" href="/items">구경하러 가기</a>
+        </div>
+        <img class="tobo-img" src="./img/Img_home_top.png" alt="장바구니를 들며 인사하는 판다 캐릭터">
+      </div>
+    </section>
+    <section class="intro-content">
+      <div class="intro-content-item">
+          <img src="./img/Img_home_01.png" alt="인기 상품을 확인하는 두 판다 캐릭터">
+          <div class="intro-content-text">
+            <h3>Hot item</h3>
+            <h2>인기 상품을 <br>확인해 보세요</h2>
+            <p>가장 HOT한 중고거래 물품을 <br>판다 마켓에서 확인해 보세요</p>
+          </div>
+      </div>
+    </section>
+    <section class="intro-content">
+      <div class="intro-content-item  intro-content-item-reverse">
+          <img src="./img/Img_home_02.png" alt="돋보기와 물음표가 그려진 박스. 검색을 표현한 일러스트">
+          <div class="intro-content-text">
+            <h3>Search</h3>
+            <h2>구매를 원하는 <br>상품을 검색하세요</h2>
```

Comment:
저는 스타일을 위해 빈 html 요소가 추가되는 걸 선호하지 않는 편입니다. 2

길이로 제한되는 케이스라면 width류의 속성을 추가하거나, 명시적으로 단락을 구분하고 싶은 경우라면 span + class 조합으로 나누면 어떨까요?


```suggestion
            <h2><span>구매를 원하는</span><span>상품을 검색하세요</span></h2>
```

Reply:
명시적으로 길이가 구분되지 않아 span.line-break를 통해 나눴습니다.